src/gui/guiFormSpecMenu.cpp

```diff
@@ -2681,7 +2686,7 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 	// Base position of contents of form
 	mydata.basepos = getBasePos();
 
-	/* Convert m_init_draw_spec to m_inventorylists */
+	/* Convert m_init_draw_spec to m_inventorylists (todo: what does that mean?) */
```

Comment:
Can I remove this comment? What is `m_init_draw_spec`?

Reply:
Lost comment. Just remove it.

Reply:
Thanks! Done.